libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -702,12 +702,12 @@ internal object MapboxRouteLineUtils {
                 resolveNumericToValue(v, routeLineColorResources)
             } ?: listOf()
         }
-    }.cacheResult(1)
```

Comment:
`resolveNumericToValue` is not cheap for long routes and without the cache it would have to be recalculated for every new route instance, even though it's constant if the `RouteLineColorResources` didn't change.

Reply:
You're right. I was looking at it from a different perspective.  I'll put the cache back.